package.json

```diff
@@ -49,16 +49,16 @@
   ],
   "license": "Apache-2.0",
   "peerDependencies": {
-    "@hapi/hapi": ">=17.9.0"
+    "@hapi/hapi": ">=19.2.0"
```

Comment:
Vision v6 still supports Hapi 18, so let's make this the same: https://github.com/hapijs/vision/blob/master/lib/index.js#L18-L20